composer.json

```diff
@@ -12,7 +12,9 @@
     },
 
     "autoload-dev": {
-        "App\\Tests\\": "tests/"
+        "psr-4": {
+            "App\\Tests\\": "tests/"
```

Comment:
Your original error message was about autoloading `DummyTest`, not `PHPUnit\Framework\TestCase`, so this was also wrong.

Reply:
Yes, silly mistake, thanks!